recipes-bsp/sgx/files/0003-default-python.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/linux/installer/common/psw/createTarball.sh b/linux/installer/common/psw/createTarball.sh
```

Comment:
Do we need this patch ? wondering rather than overriding pyhon() we could have use some inherited methods .

Reply:
Yes

Reply:
Yes